### ISSUE TITLE
how to run vcf2maf with clinvar db

### ISSUE THREAD

**Issue by songeric1107** [mskcc/vcf2maf#327]
I have vcf files which are generated using freebay, how could I add the clinvar annotation to the vcf and convert to maf?

I try to run as following

vep  --input_file JH020.soma.freebayes.vcf --output_file JH020.soma.clinc.freebayes.vcf --format vcf --species homo_sapiens  --custom /vep_ref/hg19/new/clinvar.vcf.gz,ClinVar,vcf,exact,0,CLNSIG,CLNREVSTAT,CLNSIGCONF

all the variant are intergenic_variant, which is not right

**Comment by ckandoth** [comment]
You are using VEP, not vcf2maf. But I'll try to help. If all the variants are showing as intergenic, then it is likely that you are using a transcript/reference that is a different human assembly than your VCF. E.g. hg19 VCF with hg38 VEP cache.

**Comment by songeric1107** [comment]
yes, I have tried to add the option to vcf2maf with the customed database which points to clinvar, but always failed. What is the best way to add the clinvar in vcf2maf?
